exercises/binary_converter.cpp

```diff
@@ -1,7 +1,28 @@
-/*
-  Write a program that given a number as input convert it in binary.
+#include <iostream>
+#include <cmath>
+using namespace std;
 
-  Output:
-  Insert first number: 8
-  The binary number is: 1000
-*/
+long long convertNumber(int);
+
+int main() {
+    int number, binaryCodeNumber;
+    cout << "Enter number: ";
+    cin >> number;
+    binaryCodeNumber = convertNumber(number);
```

Comment:
potresti dichiarare qui binaryCodeNumber ed usare anche `auto` come tipo in modo da prendere il tipo di ritorno di `convertNumber` e definire il tipo solo lì
```suggestion
    auto binaryCodeNumber = convertNumber(number);
```